tests/directives/test_item_attribute_directive.py

```
@@ -0,0 +1,319 @@
+"""Tests for the item-attribute directive"""
+from unittest import TestCase
+from unittest.mock import Mock, MagicMock, patch
+import logging
+
+from docutils import nodes
+from docutils.parsers.rst import directives
```

Flake8 (7.3.0)

[error] 'docutils.parsers.rst.directives' imported but unused

(F401)